### ISSUE TITLE
I didn't find the implementation of the 'linear2exp' you mentioned before

### ISSUE THREAD

**Issue by Amose-Yao** [d-li14/mobilenetv2.pytorch#10]
Several months ago you answered a question with your training command. But I didn't find the implementation of the '--lr-decay="linear2exp"' in the function 'adjust_learning_rate'.

I trained MobileNet V2 from scratch by calling
python3 imagenet.py -d /path/to/your/ImageNet/root/ -j16 --epochs=300 --arch="mobilenetv2" --gpu-id="0,1,2,3" --lr=0.045 --lr-decay="linear2exp" --gamma=0.98 --weight-decay=0.00004

Now, the best model achieves 71.79% top-1 accuracy at Epoch 249. The training is expected to finish in a few days. Hardware environment: 4-way 2080ti, Software environment: Pytorch 1.0.

_Originally posted by @lld533 in https://github.com/d-li14/mobilenetv2.pytorch/issues/2#issuecomment-454282418_

**Comment by d-li14** [comment]
First, this command line has never been mentioned by me. Second, you could find 'linear2exp' in the commit history but it is deprecated now.
